frontend/src/component/events/EventTimeline/EventTimelineEventGroup/EventTimelineEventGroup.tsx

```diff
@@ -18,19 +18,19 @@ const StyledEvent = styled('div', {
 
 interface IEventTimelineEventProps {
     group: TimelineEventGroup;
-    startDate: Date;
-    endDate: Date;
+    startTime: number;
```

Comment:
My reasoning here is that we were repeatedly converting dates to timestamps (numbers), so I think it makes sense to do it once upfront and avoid repetitive conversions.

While using `Date` could provide stricter types in this case, given the controlled nature of this component and the context of its usage, I don’t foresee any issues with using timestamps directly.

Reply:
See the diff below this comment. The left side has several `.getTime()` calls we can avoid since we already have those timestamps in the parent component. We can simply pass those down instead.

![image](https://github.com/user-attachments/assets/ffda10ec-7147-4395-a17c-b22ce9f175a3)

Reply:
Cool!